docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -119,6 +120,7 @@ The configuration file for the Rackspace provider looks like:
      username: <rackspace username>
      key: <access key>
      keyname: <ssh-key-name>
+     # FLOC-1925 Add auth_url, auth_plugin, provider
```

Comment:
If so, that's fine for now.  Cinder is the only storage driver that could work on Rackspace right now (apart from ZFS which doesn't count).  Maybe later we'll want to have something more like the `dataset` section from `agent.yml` in this file as well to separate "how to get a machine to test on" from "how to get a storage driver to test against".